.gitignore

```diff
@@ -59,3 +59,4 @@ target/
 node_modules
 
 *.min.js
+.python-version
```

Comment:
If you are using pyenv, please add this to your global config: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Wow, Just learn this good practice, Thanks!